person.rb

```diff
@@ -1,6 +1,7 @@
 require './nameable'
 require './capitalize_decorator'
 require './trimmer_decorator'
+require './rental'
 class Person < Nameable
   # initialize method
 
```

Comment:
- [ ] Getters and Setters for @rentals should be present Kindly add this 

` attr_accessor :name, :age, :rentals`

Reply:
I will set the correct attribute accessors

Reply:
Well resolved @d4nQw3rty 👍🏽